tests/modules/token_embedders/pretrained_transformer_embedder_test.py

```diff
@@ -220,7 +220,7 @@ def test_unfold_long_sequences(self):
         assert (unfolded_embeddings_out == unfolded_embeddings).all()
 
     def test_encoder_decoder_model(self):
-        token_embedder = PretrainedTransformerEmbedder("facebook/bart-large", sub_module="encoder")
+        token_embedder = PretrainedTransformerEmbedder("bart-large", sub_module="encoder")
```

Comment:
Did they rename this back? I'm so confused because I think I had to add the `facebook/` part a few weeks ago.

Reply:
Looks like they did. I pinned the transformers version more tightly in this PR to make this work.